pkg/log/log_context.go

```diff
@@ -0,0 +1,31 @@
+package log
+
+import (
+	"github.com/gobuffalo/buffalo"
+	"github.com/sirupsen/logrus"
+)
+
+const logEntryKey string = "log-entry-context-key"
```

Comment:
no need to specify the type just `const logEntryKey = "..."`

Reply:
we can also use custom type for safety so it is extractable only for us

Reply:
@michalpristas I had the same thought but the `Set` method accepts `(string, interface{})`. I didn't know of a way around that.

Reply:
@penDerGraft it should accept it because a constant with no type and quotes is just a string as well.

Reply:
ooh this is buffalo context, not the classic one. please ignore me

Reply:
Ops. I didn't read @michalpristas's comments. Yes, this is a buffalo middleware :) -- if we end up removing buffalo for the API part some time in the future then we'd use the regular convention of making a special type for context keys.